class-01-responsive-mobile-first/pair-assignment/starter-code/styles/icons.css

```diff
@@ -36,6 +36,7 @@
 	content: "\e909";
 }
 .icon-power::before {
+	float: right;
```

Comment:
it would be better to add this rule in one of your CSS files rather than altering vendor code.